Makefile

```diff
@@ -101,9 +111,15 @@ else
 CONTROLLER_GEN=$(shell which controller-gen)
 endif
 
-create-installers: set-image
+create-installers-china: modify-base-kustomize-china
+	kustomize build config/installers/rolebasedcreds > $(INSTALLER_PATH)/rolebased/installer_china.yaml
+	kustomize build config/installers/rolebasedcreds/namespaced > $(INSTALLER_PATH)/rolebased/namespaced/operator_china.yaml
```

Comment:
I think we should have a separate folder for the Chinese releases so people don't accidentally try to deploy them elsewhere. Perhaps just a subfolder.

Reply:
done